src/lib/types/order.ts

```diff
@@ -0,0 +1,17 @@
+
+// Add new type for orders
+export interface EngineOrder {
+    orderId: string;
```

Comment:
```suggestion
    exchangeOrderId: string;
```
So it's more obvious that it's how the exchange identified the order